numpy/_core/src/npysort/highway_qsort_16bit.dispatch.cpp

```diff
@@ -1,5 +1,8 @@
-#include "highway_qsort.hpp"
+#ifdef NPY_HAVE_VSX
+    #define HWY_HAVE_FLOAT16 0
```

Comment:
It's the other way around: Highway already sets HAVE_FLOAT16 to 0 to inform user code when float16 cannot be used. VQSort counts as 'user code'.
What we could do is define VQSORT_ENABLED to 0 in this file if !HWY_HAVE_FLOAT16.

Reply:
Thank you for clarifying it, I made some changes to provide a scalar looping fallback when `HWY_HAVE_FLOAT16` is false, and the build passed after.